rds-apb/ansibleapp.yml

```diff
@@ -0,0 +1,70 @@
+id: 84e8173e-5ced-4489-9b4d-aac7e779c47e
+name: ansibleplaybookbundle/rds-apb
```

Comment:
PR #41 has an rds-postgres-apb. I've mentioned going forward it doesn't really make sense to have 3 (or more...) apb's for rds as it should be possible to handle all scenarios with just one so I think it would make sense to figure out how we do that, but for the demo the other PR will do.